week-2/Homework/mandatory/3-playing-computer.js

```diff
@@ -7,36 +7,36 @@
   Answer the following questions:
 
   1. This program throws an error. Why? (If you can't find it, try executing it).
-  2. Remove the line that throws the error.
+  2. Remove the line that throws the error. 
   3. What is printed to the console?
   4. How many times is "f1" called?
   5. How many times is "f2" called?
   6. What value does the "a" parameter take in the first "f1" call?
   7. What is the value of the "a" outer variable when "f1" is called for the first time?
 */
 
-let x = 2;
-let a = 6;
+let x = 2;// variable x assigned to 2
+let a = 6;// variable a assigned to 6
 
-const f1 = function(a, b) {
-  return a + b;
+const f1 = function(a, b) {//f1 is a function with a prameters  a and b
+  return a + b;// sums a and returns the result  However this would return an error because 'b' is not defined
```

Comment:
Are you sure that `b` isn't defined? Take another look at the arguments to the function :)